recipes/limereport/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+import os
+
+
+class LimereportConan(ConanFile):
+    name = "limereport"
+    description = "Report generator for Qt Framework"
+    homepage = "https://limereport.ru"
+    topics = ("limereport", "pdf", "report","qt")
+    license = "LGPL-3.0", "GPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_zint": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_zint": False,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "8",
+            "clang": "8",
+            "apple-clang": "9.1",
+        }
+
+    @property
+    def _qt_version_major(self):
+        return Version(self.dependencies["qt"].ref.version).major
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        # QString included in Irglobal.h and Limereport expects be running Qt on customer side
+        self.requires("qt/6.7.1", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("qt/[>=6.0 <7]", transitive_headers=True, transitive_libs=True)
```

We are actually enabling version ranges for Qt. It should avoid not only conflicts, but improve the build time when packaging new versions and dropping older versions.

Reply:
@maksim-petukhov Qt5 can be applied by using [replace_requires](https://docs.conan.io/2/reference/config_files/profiles.html#replace-requires) :)

Reply:
@maksim-petukhov I just updated to support both majors. Thank you!